.storybook/stories/kitchenSink.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
I was getting a "React is an UMD module" error in vscode, which this seemed to fix.